interface/Cargo.toml

```
@@ -42,10 +42,12 @@ strum = "0.24"
 strum_macros = "0.24"
```

We should add this as a dev-dependency with the bincode feature enabled so that the tests run by default
```suggestion
solana-system-interface = { path = ".", features = ["bincode] }
strum_macros = "0.24"
```